index.js

```diff
@@ -176,18 +176,18 @@ function getExistingPath (command, opts) {
 
 module.exports._getNpmCache = getNpmCache
 function getNpmCache (opts) {
-  return which(opts.npm).then(npmPath => {
-    const args = ['config', 'get', 'cache', '--parseable']
+  return which('node').then(nodePath => {
+    const args = [opts.npm, 'config', 'get', 'cache', '--parseable']
```

Comment:
Does this actually still work if someone does `--npm=npm`? 🤔  That's why this code was doing this. The `opts.npm` option can be a path to the npm script itself _or_ to the name of the binary in `$PATH`.